generator/generator_config.textproto

```diff
@@ -385,6 +385,14 @@ service {
   retryable_status_codes: ["kDeadlineExceeded", "kUnavailable"]
 }
 
+# Language
+service {
+  service_proto_path: "google/cloud/language/v1/language_service.proto"
+  product_path: "google/cloud/language"
+  initial_copyright_year: "2022"
+  retryable_status_codes: ["kUnavailable"]
```

Comment:
https://github.com/googleapis/googleapis/blob/2b37083027fcfa408ee75eee4264cad7c2de0a67/google/cloud/language/v1/language_grpc_service_config.json#L35-L38

I dont remember where we landed with the AIP vs. service config json rules. But I think if this one doesn't have `kDeadlineExceeded`, then no service should have it...

Reply:
I don't think we reached a decision, but I think (a) we should never have `kDeadlineExceeded`, and (b) this is easy to change after the fact.

Reply:
Ack.